ldes-server-infra-mongo/mongo-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/infra/mongo/fragment/entity/LdesFragmentEntity.java

```diff
@@ -77,12 +79,13 @@ public List<FragmentPair> getFragmentPairs() {
 	}
 
 	public static LdesFragmentEntity fromLdesFragment(LdesFragment ldesFragment) {
+		ViewName localViewName = ldesFragment.getViewName();
 		return new LdesFragmentEntity(ldesFragment.getFragmentId(),
 				ldesFragment.getFragmentPairs().isEmpty(),
-				ldesFragment.getViewName(),
+				localViewName.toString(),
 				ldesFragment.getFragmentPairs(), ldesFragment.isImmutable(),
 				ldesFragment.isSoftDeleted(), ldesFragment.getParentId(), ldesFragment.getImmutableTimestamp(),
-				ldesFragment.getNumberOfMembers(), ldesFragment.getRelations(), ldesFragment.getCollectionName());
+				ldesFragment.getNumberOfMembers(), ldesFragment.getRelations(), localViewName.getCollectionName());
```

Comment:
Really liked that you left this as is (since otherwise we would need another Mongock :-D)
Maybe we could log it as an issue to adapt the LdesFragmentEntity to have an attribute ViewName and remove the collectionName and viewName strings.